cmd/fluent-manager/main.go

```diff
@@ -230,10 +230,21 @@ func main() {
 		}
 	}
 
-	if envs, err := godotenv.Read("/fluent-operator/fluent-bit.env"); err == nil {
+	// CONTAINER_ROOT_DIR provides a volume+mount for the Fluentbit DaemonSet so that
+	// Fluentbit can read container logs from nodes.
+	// Falls back to legacy file-based configuration for backward compatibility
+	if envLogPath := os.Getenv("CONTAINER_LOG_PATH"); envLogPath != "" {
+		logPath = envLogPath
+	} else if envs, err := godotenv.Read("/fluent-operator/fluent-bit.env"); err == nil {
 		logPath = envs["CONTAINER_ROOT_DIR"] + "/containers"
 	}
 
+	// If neither the env var or file based config is available, fall back to the
+	// default path for containerd/CRI-O
```

Comment:
```suggestion
	if envLogPath := os.Getenv("CONTAINER_LOG_PATH"); envLogPath != "" {
		logPath = envLogPath
	} else if envs, err := godotenv.Read("/fluent-operator/fluent-bit.env"); err == nil {
		logPath = envs["CONTAINER_ROOT_DIR"] + "/containers"
	}

```

I'm not a fan of obvious inline comments. The code should be self-explanatory, which it is in this case. So no need for comments.